beetsplug/subsonicupdate.py

```diff
@@ -78,10 +86,26 @@ def loaded(self):
             'v': '1.15.0',  # Subsonic 6.1 and newer.
             'c': 'beets'
         }
+
+        if port == 443:
+            schema = 'https://'
+
         if contextpath == '/':
             contextpath = ''
-        url = "http://{}:{}{}/rest/startScan".format(host, port, contextpath)
+
+        if not url:
+            url = "{}{}:{}{}/rest/startScan" \
+                .format(schema, host, port, contextpath)
+        else:
+            url += '/rest/startScan'
+
         response = requests.post(url, params=payload)
 
-        if response.status_code != 200:
-            self._log.error(u'Generic error, please try again later.')
+        if response.status_code == 403:
+            self._log.error(u'Forbidden, invalid permissions')
+        elif response.status_code == 404:
+            self._log.error(u'Not Found, resource cannot be found')
```

Comment:
The message seems a little redundant. Maybe this would be better off with the generic error message below, just with the status code included?